rustfmt.toml

```diff
@@ -1,51 +1,56 @@
 max_width = 100
```

Comment:
@boxdot why even have a `rustfmt.toml` file at all? aren't the default settings fine?

Reply:
This is from the defaults.  As I understand these evolve so it's better to check it in so you don't get too many spurious formatting changes.